src/styles/LandingPageStyles/index.less

```diff
@@ -15,6 +15,7 @@
 
 .landing-main-title {
   font-family: 'Montserrat', cursive !important;
```

Comment:
Paul & Edd also mentioned trying to stay away from using !important tags, so this is something as a team we should look into and discuss as I know it also used on previously written code.